torch/testing/_internal/common_optimizers.py

```diff
@@ -385,8 +385,6 @@ def optim_error_inputs_func_adagrad(device, dtype):
     return error_inputs
 
 
-# TODO: consider tensor LR! See multi_tensor_optimizer_configs in test_optim.py --> tensor LR should work
-# with all implementation code paths...
```

Comment:
This is going to require more work as foreach nherently does not support tensor lr, even on CUDA/CPU, for performance reasons.